ui/app/rif/pages/domainsDetailPage/domainDetailActive/domainDetailActive.js

```diff
@@ -98,22 +98,22 @@ const styles = {
 }
 
 class DomainsDetailActiveScreen extends Component {
-	static propTypes = {
+  static propTypes = {
     addNewChainAddress: PropTypes.func.isRequired,
     setNewResolver: PropTypes.func.isRequired,
     deleteChainAddressForResolver: PropTypes.func.isRequired,
-		setAutoRenew: PropTypes.func.isRequired,
+    setAutoRenew: PropTypes.func.isRequired,
     getUnapprovedTransactions: PropTypes.func,
     domain: PropTypes.object.isRequired,
-		domainName: PropTypes.string.isRequired,
-		address: PropTypes.string.isRequired,
-		content: PropTypes.string.isRequired,
-		expirationDate: PropTypes.string.isRequired,
-		autoRenew: PropTypes.bool.isRequired,
-		ownerAddress: PropTypes.string.isRequired,
+    domainName: PropTypes.string.isRequired,
```

Comment:
Be careful with the identation here, probably you are using visual studio code to work and that changes the identation to antoher different from the nifty standar.